code/modules/clothing/rogueclothes/armor/armor.dm

```diff
@@ -1072,7 +1085,7 @@
 	desc = "A large shirt with heavy padding meant to be used below armor."
 	icon_state = "fancygamb"
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS|VITALS
-	armor = list("blunt" = 60, "slash" = 40, "stab" = 50, "piercing" = 25, "fire" = 0, "acid" = 0)
+	armor = list("blunt" = 60, "slash" = 40, "stab" = 50, "bullet" = 20, "fire" = 0, "acid" = 0)
```

Comment:
This is effectively the exact same stat block as our heavy gambesons. While I'm a fan of stylepoints, It would be better to adjust the stats. If it's meant to be better for fencing, raise it's stab to 60 and drop blunt. Maybe tweak slash, maybe not. up to you.